### ISSUE TITLE
Santroller not recognized in Fortnite

### ISSUE THREAD

**Issue by jaronlucas** [InvoxiPlayGames/FestivalInstrumentMapper#55]
I have mapped my santroller, and it worked one time in the program to be recognized as a RB controller, but it has not worked since. When it was working, however, it kept saying "swap device". I tried pressing an input on my controller to see if that would make the message go away and that unfortunately did not do the trick. But now, it's not even being recognized in the game period. I did not change anything program-wise, which is weird.

**Comment by sanjay900** [comment]
From what i understand, the way around the swap device problem is to use the keyboard and mouse for most of the menus and then to swap to the guitar when it says swap device.

**Comment by jaronlucas** [comment]
> From what i understand, the way around the swap device problem is to use the keyboard and mouse for most of the menus and then to swap to the guitar when it says swap device.

Yeah, unfortunately that's what I have been doing. It still has been saying swap device the second I'm not pressing a fret or strumming the strum bar.

**Comment by gozfrt** [comment]
You have to turn off "Ignore Controller Inputs" on Fortnite

**Comment by jaronlucas** [comment]
I did turn that off too

**Comment by sanjay900** [comment]
just FYI, you no longer actually need FIM to use a santroller with festival, there is a mode that emulates just enough of a controller to make FNF happy without needing auth in the latest version

**Comment by jaronlucas** [comment]
> just FYI, you no longer actually need FIM to use a santroller with festival, there is a mode that emulates just enough of a controller to make FNF happy without needing auth in the latest version

Thanks Sanjay! My santroller seller has no updated his program yet though unfortunately..
